devel/grpc/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup  github 1.0
+PortGroup  cxx11 1.1
+
+github.setup    grpc grpc 1.9.1 v
+categories      devel
+maintainers     nomaintainer
```

Comment:
Do you have any reason for not taking over maintainership? Non-maintained ports are highly undesirable as that means you won't get any bug reports or build failures reported, while we have nobody to turn to when something fails. Ideally you would put both your github handle and obfuscated email here + `openmaintainer` keyword (described in the guide).

Reply:
I do not know how long I will be actively using gRPC. Hence I do not want to commit to maintaining this port right now.  Apart from the build patch, it's a straightforward port as far as I can tell.